packaging/windows/WixUI_HK.wxs

```diff
@@ -11,7 +11,7 @@
          <Property Id="WixUI_Mode" Value="InstallDir" />
          <Property Id="ARPNOMODIFY" Value="1" />
          <Property Id="ARPNOREPAIR" Value="1" />
-         <Property Id="WIXUI_EXITDIALOGOPTIONALTEXT" Value="CodeReady Containers is installed in your computer. Run the command 'crc.exe setup' to setup your environment." />
+         <Property Id="WIXUI_EXITDIALOGOPTIONALTEXT" Value="CodeReady Containers is installed in your computer. Open PowerShell or Command Prompt and run the command 'crc.exe setup' to setup your environment." />

```

Comment:
I would change this to always say 'Please reboot your computer'

Reply:
what about the `crc setup` message? the tray would show a message box saying this after the reboot though

Reply:
There was condition missing for the `<SheduleReboot>` element in the next commits and the prompt was not shown, fixed this in the latest push

Reply:
@guillaumerose Do you think the new message is good, with the reboot prompt at the end?